docs/navda/navda-intro.mdx

```diff
@@ -1,18 +1,18 @@
 ---
 sidebar_position: 1
 ---
-# Intro to Navda
+# Introduction to Navda
 
 :::caution
 ### 🚧 Under Construction 🚧
 This section of documentation is still under construction — it is not formalized and may change.
-Help contribute to it on our [GitHub](https://github.com/Avdan-OS/docsite/tree/main/docs/navda/navda-intro.mdx).
+Help contribute on our [GitHub](https://github.com/Avdan-OS/docsite/tree/main/docs/navda/navda-intro.mdx).
 :::
 
 > 'Navda' — isn't that 'Avdan' backwards ? 
 
 
-*Navda* is AvdanOS' display server.
+*Navda* is AvdanOS display server.
```

Comment:
Grammatically, `AvdanOS' display server` (or `AvdanOS's display server`) shows that the display server belongs to (or is a part of) to AvdanOS.